### ISSUE TITLE
Missing Document

### ISSUE THREAD

**Issue by gracewm** [intel-iot-devkit/pneumonia-classification#1]
When I run the code in pneumonia_classification_jupyter.ipynb, I got the following error:

Exception: Path to the model ../resources/FP32/model.xml doesn't exist or it's a directory

In the resources directory, there's no FP32 folder.  However, there's  model/model.pb exists.
How can I get the model.xml file.

![issue](https://user-images.githubusercontent.com/12513863/73984154-59864c00-497b-11ea-9d6f-0a3c396d8058.png)

**Comment by MandipJ** [comment]
@gracewm 
Did you run **setup.sh** script as mentioned in the Readme?
It will convert the model to xml and bin files required by the application.

**Comment by gracewm** [comment]
I am working on Windows 10.  So, I executed the following command:

!python C:/OpenVino/openvino/deployment_tools/model_optimizer/mo_tf.py -m $BASE_DIR/resources/model/model.pb --input_shape=[1,224,224,3] --data_type FP32 -o $BASE_DIR/resources/FP32 --mean_values [123.75,116.28,103.58] --scale_values [58.395,57.12,57.375]

Got the following result:

[ SUCCESS ] Generated IR model.
[ SUCCESS ] XML file: C:\Users\USER\Documents\pneumonia-classification-master/resources/FP32\model.xml
[ SUCCESS ] BIN file: C:\Users\USER\Documents\pneumonia-classification-master/resources/FP32\model.bin
[ SUCCESS ] Total execution time: 774.89 seconds. 

However, after running the following command, I got the following error message:
!python pneumonia_classification_jupyter.py -m ../resources/FP32/model.xml -o ../output

Traceback (most recent call last):
  File "pneumonia_classification_jupyter.py", line 383, in <module>
    main()
  File "pneumonia_classification_jupyter.py", line 374, in main
    save_img(output_file, img1, file_format='jpeg')
  File "pneumonia_classification_jupyter.py", line 122, in save_img
    img.save(path, format=file_format, **kwargs)
  File "C:\Users\USER\Anaconda3\lib\site-packages\PIL\Image.py", line 2081, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: '../output/PNEUMONIA\\person1946_bacteria_4874.jpeg'

In the output folder, there are result.txt file containing probability and an empty file of stats.txt.
No output image files saved.

**Comment by gracewm** [comment]
Instead of running pneumonia_classification_jupyter.py on the notebook cell, I executed the following command:

!python pneumonia_classification_jupyter.py -m ../resources/FP32/model.xml -o ../output

After running the above command, got result.txt and stats.txt file, which are not empty.  Also, after copying PNEUMONIA folder to the output directory, I also got image files, where class_activation heatmap is overlaid over the X-ray image in case of pneumonia.  The wider the yellow color area, the higher the probability of pneumonia is. The file not found error is caused as there are no base X-ray image files to be overlaid. Looks like in the pneumonia_classification_jupyter.py file, there is no code copying PNEUMONIA folder to the output directory (I manually copied the folder).
